app/components/Channels/ChannelSummaryListItem.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box, Flex } from 'rebass'
+import { Card, Heading, Text } from 'components/UI'
+import { ChannelCapacity, ChannelMoreButton, ChannelStatus } from 'components/Channels'
+import withEllipsis from 'components/withEllipsis'
+
+const ClippedHeading = withEllipsis(Heading.h1)
+const ClippedText = withEllipsis(Text)
+
+const ChannelSummaryListItem = ({
+  channelId,
+  channelName,
+  channelPubKey,
+  cryptoCurrency,
+  cryptoCurrencies,
+  localBalance,
+  remoteBalance,
+  status,
+  setCryptoCurrency,
+  showChannelDetail,
+  ...rest
+}) => (
+  <Card {...rest}>
+    <Flex>
+      <Box width={8 / 20}>
+        <ChannelStatus status={status} />
+        <ClippedHeading my={1}>{channelName || channelPubKey.substring(0, 10)}</ClippedHeading>
+        <ClippedText fontSize="xs">{channelPubKey}</ClippedText>
+      </Box>
+
+      <ChannelCapacity
+        localBalance={localBalance}
+        remoteBalance={remoteBalance}
+        cryptoCurrency={cryptoCurrency}
+        cryptoCurrencies={cryptoCurrencies}
+        setCryptoCurrency={setCryptoCurrency}
+        width={1 / 2}
+        pl={4}
+        pr={4}
+      />
+
+      <Flex width={2 / 20} alignSelf="stretch">
```

Comment:
I think `alignSelf="stretch"` can be safely removed as it basically a default

Reply:
removed.